lib/justifi/justifi_response.rb

```diff
@@ -80,11 +80,18 @@ class JustifiResponse
     # object.
     def self.from_net_http(http_resp)
       resp = JustifiResponse.new
-      resp.data = JSON.parse(http_resp.body, symbolize_names: true)
       resp.http_body = http_resp.body
       resp.success = http_resp.is_a? Net::HTTPSuccess
-      resp.error_message = resp.data.dig(:error, :message)
-      resp.error_details = resp.data.dig(:error)
+
+      begin
+        resp.data = JSON.parse(http_resp.body, symbolize_names: true) if http_resp.body
+      rescue JSON::ParserError
+        resp.data = nil
+      end
+
+      resp.error_message = resp.data&.dig(:error, :message)
+      resp.error_details = resp.data&.dig(:error)
```

Comment:
I think for these we can have a default error message, something like. wdyt?
```suggestion
      resp.error_message = resp.data&.dig(:error, :message) || "Unexpected request response"
      resp.error_details = resp.data&.dig(:error) || "internal_server_error"
```